websockets/feeds/cinema.go

```diff
@@ -278,12 +278,32 @@ func getInfoRawFile(url string) (cv cinemaVideo, err error) {
 	return
 }
 
+func getInfoYoutube(url string) (cv cinemaVideo, err error) {
+    args := []string{
+		"-j", url,
+    }
+	var out []byte
+	out, err = util.ExecBinary("youtube-dl", args, 8000*time.Millisecond)
```

Comment:
Unneeded with https://github.com/iawia002/annie, we should stay away from executing binaries from the server if we can help it, I think.